src/Storages/MergeTree/MergeTreeData.cpp

```diff
@@ -8083,12 +8084,11 @@ std::pair<MergeTreeData::MutableDataPartPtr, scope_guard> MergeTreeData::cloneAn
         dst_part_storage = src_part_storage->freezeRemote(
```

Comment:
Shouldn't we in code above throw error with error code from `reserve` unexpected result?